guides/contribution-guide.md

```diff
@@ -19,6 +19,10 @@ We would love for you to contribute to Cashmere and be part of the community mak
 4.  Clone your fork.
     Recommendation: name your git remotes `upstream` for `@healthcatalyst/cashmere`
 5.  From the root of the project, run `npm install`.
+6.  Running `npm run build` will build the entire project
+7.  While developing you can use angular cli's `--watch` to automatically build a project when the source files change
```

Comment:
It may be wise to mention they also need to run `ng serve` in addition to `npm run build`

Reply:
I'll add that but I think that is getting into documenting `@angular/cli` itself. The reason i'd avoid putting that in here is that we shouldn't bloat our own docs with other tooling docs. If a dev doesn't know a tool then then can easily look up the docs to figure out how to use it.